themes/ddbasic/sass/components/kss.scss

```diff
@@ -0,0 +1,80 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?